pnytter/nitter_parser/profiles.py

```diff
@@ -46,7 +46,10 @@ def _get_profile_fullname(self) -> str:
         return self.soup.find("a", class_="profile-card-fullname").text
 
     def _get_profile_biography(self) -> str:
-        return self.soup.find("div", class_="profile-bio").text
+        if self.soup.find("div", class_="profile-bio"):
+            return self.soup.find("div", class_="profile-bio").text
+        else:
+            return ""
```

Comment:
Just some style refactor, to avoid re-calling the `soup.find()` (:

```suggestion
        div = self.soup.find("div", class_="profile-bio")
        return div.text if div else ""
```